typescript/infra/scripts/warp-routes/deploy-warp-monitor.ts

```diff
@@ -15,11 +16,10 @@ async function main() {
     .string('filePath')
     .parse();
 
-  await runWarpRouteHelmCommand(
-    HelmCommand.InstallOrUpgrade,
-    'mainnet3',
-    filePath,
-  );
+  await assertCorrectKubeContext(getEnvironmentConfig('mainnet3'));
```

Comment:
nit: thought on a enum for this?

Reply:
+1, would be useful in other places

Reply:
Spent a bunch of time on this just to hit a bunch of circular deps in infra

Would be happy to add this but I'm gonna call it for now, a bit rabbithole-y without a huge payoff